package.json

```diff
@@ -10,8 +10,10 @@
     "license": "Apache-2.0",
     "scripts": {
         "start": "npm run node-hid:prebuild-install && electron .",
-        "test": "TZ=UTC vue-cli-service test:unit",
-        "test:jenkins": "TZ=UTC vue-cli-service test:unit --coverage",
+        "test": "npm run jestClearCache && TZ=UTC vue-cli-service test:unit --testTimeout=30000",
+        "test:jenkins": "npm run jestClearCache && TZ=UTC vue-cli-service test:unit --coverage --testTimeout=30000",
```

Comment:
is there any test taking longer than 10s?

Reply:
there was, complex UI page unit tests such as `FormPersistentDelegationRequestTransaction` could take more than 15s _depending on the running machine_.